source/guide_invoiceninja.rst

```diff
@@ -136,8 +127,8 @@ To automatically send recurring invoices and reminders we need to setup some cro
 
 ::
 
- 0 8 * * * /usr/bin/php /var/www/virtual/$USER/invoiceninja/artisan ninja:send-invoices
```

Comment:
Are you sure about that? The prevous `[isabell@stardust ~]$ mv ninja/* html/` step installs invoice ninja into `/var/www/virtual/$USER/html`. Where is the `artisan` script supposed to come from?

Reply:
Mea culpa, I forgot the html/. Will change that

Reply:
I double checked by doing the whole thing on ninjatmp@gale.uber.space. Feel free to have a look. Will remove the account after successful merge